package.json

```diff
@@ -25,18 +25,7 @@
     "react": "^18",
     "react-dom": "^18",
     "react-google-charts": "^4.0.1",
-    "framer-motion": "^11.2.10",
-    "@radix-ui/react-checkbox": "^1.1.0",
-    "@radix-ui/react-dialog": "^1.1.1",
-    "@radix-ui/react-dropdown-menu": "^2.1.1",
-    "@radix-ui/react-label": "^2.1.0",
-    "@radix-ui/react-select": "^2.1.1",
-    "@radix-ui/react-slot": "^1.1.0",
-    "@radix-ui/react-tooltip": "^1.1.1",
-    "@tanstack/react-table": "^8.17.3",
-    "class-variance-authority": "^0.7.0",
-    "clsx": "^2.1.1",
-    "lucide-react": "^0.396.0",
```

Comment:
Is there a reason why these packages were removed from package.json? Wondering if it might cause some build errors for devs who install using `npm install` (I don't think it effects the Vercel build though, that should be using package-lock.json via `npm ci`)